_includes/head.html

```diff
@@ -77,7 +97,7 @@
   <!-- End Piwik Code -->
   {% endif %}
 
-  <link rel="alternate" type="application/rss+xml" title="{{ site.title }}" href="/feed/">
+  <link rel="alternate" type="application/rss+xml" title="{{ site.title }}" href="/feed/" />
```

Comment:
Some websites using chameleon don't have feeds at all

Reply:
Now it is optional with `enable_feed` option.